src/ui/components/molecules/GalleryForm/GalleryForm.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import styles from './GalleryForm.module.scss';
+import TextInput from '../../atoms/TextInput/TextInput';
+import type { IProps } from './Types/IProps';
+
+function GalleryForm({ onInputChange = () => {}, size = 'medium', ...props }: IProps) {
+  const handleChange = (field: string) => (event: React.ChangeEvent<HTMLInputElement>) => {
+    onInputChange(field, event.target.value);
+  };
+
+  return (
+    // eslint-disable-next-line react/jsx-props-no-spreading
```

Comment:
please create a ticket, to remove the rule in the styles guide 
  // eslint-disable-next-line react/jsx-props-no-spreading